### PR TITLE
Feature/ext 31/localization support

### PR DESCRIPTION
**Pull request by bradleySuira**
# :books: Linear Ticket :books: #ext-31
This PR includes:
- Localization support 
- Base English and Spanish keys
- Localization switch inside settings page

### CODE REVIEW

File: src/locales/en/common.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "type": "Type",
+    "continue": "Continue",
+    "next": "Next",
+    "back": "Back",
+    "new": "New",
+    "status": "Status",
+    "start": "Start",
+    "end": "End",
+    "completed": "Completed",
+    "advanved": "Advanved",
```
Comment (stephien) [inline src/locales/en/common.json:12]: Hey @bradleySuira , both the code and word in English have a typo. We’ll fix these two typos in the common.json file using Crowdin, but I think you need to fix it here and other places.

File: src/locales/en/pages.json [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        "assets": "Assets",
+        "activity": "Activity",
+        "nfts": {
+            "zero": "NFTs (0)",
+            "one": "NFTs",
+            "other": "NFTs (%{count})"
+        },
+        "nftActivityDescription": "Once you start using your NFTs you will see the activity here",
+        "nftDescription": "Once you start owning NFTs with accounts in your Liquality wallet you will see them here",
+        "exploreNFT": "Explore NFTs on Polygon, Ethereum and Solana",
```
Comment (stephien) [inline src/locales/en/pages.json:111]: Shouldn’t Arbitrum also be mentioned here?

Comment (bradleySuira) [reply src/locales/en/pages.json:111]: thanks @stephien in this case I just copy the actual content from the wallet texts but if we don't need to mention Arbitrum I think that I can remove it just let me know.

File: src/locales/en/common.json (hunk 1 above)
Comment (bradleySuira) [reply src/locales/en/common.json]: thanks @stephien fixed now in all files

File: vue.config.js [hunk 3]
```diff
@@ -82,9 +88,6 @@ module.exports = {
       manifestTransformer: (manifest) => {
         manifest.content_security_policy =
           "script-src 'self' 'unsafe-eval' 'unsafe-inline' https://cdn.segment.com 'sha256-ZgDy59Dh4jH9g/vcPYFpoQ1wumB4IdPEOS1BJc08i+Y='; object-src 'self';"
-        manifest.externally_connectable = {
```
Comment (monokh) [inline vue.config.js:85]: why was it removed? it won't affect anytrhing?

**Review by monokh — APPROVED** [review]
LGTM

File: vue.config.js (hunk 3 above)
Comment (bradleySuira) [reply vue.config.js:85]: yes in this case we are not using anymore the ledger bridge so we don't need this settings

**Pull request merged**